pkg/oc/oc.go

```diff
@@ -0,0 +1,18 @@
+package oc
+
+import (
+	"log"
+
+	"github.com/openshift-pipelines/release-tests/pkg/cmd"
+	resource "github.com/openshift-pipelines/release-tests/pkg/config"
+)
+
+// Create resources using oc command
+func Create(path_dir, namespace string) {
+	log.Printf("output: %s\n", cmd.MustSucceed("oc", "create", "-f", resource.Path(path_dir), "-n", namespace).Stdout())
```

Comment:
better to return the output.

Reply:
As we do all our assertions inside `cmd.Mustsucced`, so I see no value in returning output (atleast for my use case)

Reply:
yes `cmd.Mustsucced` will assert for exit code `0` and returns result